dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/data/upload/DataUploadRunnableTest.kt

```diff
@@ -127,7 +127,7 @@ internal class DataUploadRunnableTest {
         names = ["DISCHARGING", "NOT_CHARGING"],
         mode = EnumSource.Mode.INCLUDE
     )
-    fun `doesn't send batch when battery is low and unplugged`(
+    fun `doesn't send batch when battery is low and charging`(
```

Comment:
If the battery is low and charging, it should send the batch. I don't think the renaming was expected here.